src/lib/models/scheduled-notification.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Model = require('./model');
+
+const validators = require('./validators');
+
+class ScheduledNotification extends Model {
+  get recipient () { return this._recipient; }
+  set recipient (value) {
+    validators.assertNullableString(value);
```

Comment:
I think the recipient here is either null or an email address - would we wish to add validation for email here?

Reply:
I don't think so without a wider change to the notifications. I found entries in the database where the recipient is 'n/a' so think it might be risky to enforce the email validation here.